config/rollup.config.aliases.js

```diff
@@ -0,0 +1,12 @@
+import memory from 'rollup-plugin-memory';
+import rollupConfig from './rollup.config';
+
+export default Object.assign({}, rollupConfig, {
+	plugins: [
+		memory({
+			path: 'src/preact',
+			contents: `import { h } from './preact';export * from './preact';export { h as createElement };`
```

Comment:
I'm fine leaving it in memory for now, just was riffing.
I'll merge this, but I'm currently _slaving_ over a really annoying bug that'll likely block further releases until it's fixed :( :( :(